modules/ROOT/pages/layers.adoc

```diff
@@ -23,7 +23,6 @@ This is an application property, not a server property.
 When this property is used, nodes automatically appear in the specified layer. If the layer doesn't exist it is created.
 
 [NOTE]
```

Comment:
@asimons-mule We have another note very close to this one. Are we able to combine these or remove a Note designation?

Reply:
Good point. The first note: "This is an application property, not a server property" can just become the final sentence of the paragraph above it.